cluster/manifests.go

```diff
@@ -21,7 +21,7 @@ type Manifests interface {
 	// spec given.
 	UpdateDefinition(def []byte, container string, newImageID image.Ref) ([]byte, error)
 	// Load all the resource manifests under the path given
-	LoadManifests(paths ...string) (map[string]resource.Resource, error)
+	LoadManifests(base, first string, rest ...string) (map[string]resource.Resource, error)
```

Comment:
What do `base`, `first`, `rest` mean?

Reply:
Worth a comment, do you think?

Reply:
> Worth a comment, do you think?
I do. That or a change to `baseDir` so it's more obvious that it's a directory. The problem with `string` is you can put anything in it and it's basically just `[]byte` with the implication it might be readable when printed.

Reply:
> The problem with string is you can put anything in it

I shall use the filesystem path types from Go's standard lib. OK done.